notebooks/3_KernelMethods.ipynb

```diff
@@ -1272,8 +1281,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "Y_krr_train = krr.transform(X_train, K=K_train)\n",
-    "Y_krr_test = krr.transform(X_test, K=K_test)"
+    "Y_krr_train = krr.predict(X_train)\n",
+    "Y_krr_test = krr.predict(X_test)"
    ]
```

Comment:
^^

Reply:
Still needs to be fixed.

Reply:
Done.

Reply:
What am I supposed to "see above"?

Reply:
![image](https://user-images.githubusercontent.com/56018055/100072521-a371ed00-2e3c-11eb-993c-b8d11e6c9d90.png)